Library/Formula/gambit-scheme.rb

```diff
@@ -32,7 +34,7 @@ def install
     # Don't enable this when using clang, per configure warning.
     args << "--enable-single-host" unless ENV.compiler == :clang
 
-    args << "--enable-shared" if build.include? "enable-shared"
+    args << "--enable-shared" if build.with? "shared"
 
     if ENV.compiler == :clang
```

Comment:
Oh, I missed @nijikon's comment above.

Thanks for sticking with this, and figuring it out!